Update aws-batch CE page with EFS and FSx mount path details (#272)
seqeralabs/nf-tower-docs: docs/compute-envs/aws-batch.md

15. Enter any additional **Allowed S3 buckets** that your workflows require to read input data or write output data. The **Pipeline work directory** bucket above is added by default to the list of **Allowed S3 buckets**.

> 16. To use **EFS**, you can either select **Use existing EFS file system** and specify an existing EFS instance, or select **Create new EFS file system** to create one. For new and existing EFS file systems, you will need to specify `<your_EFS_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide).

Author: ```suggestion
16. To use **EFS**, you can either select **Use existing EFS file system** and specify an existing EFS instance, or select **Create new EFS file system** to create one. If you intend to use the EFS file system as your work directory, you will need to specify `<your_EFS_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide).
```

> 16. To use **EFS**, you can either select **Use existing EFS file system** and specify an existing EFS instance, or select **Create new EFS file system** to create one. If you intend to use the EFS file system as your work directory, you will need to specify `<your_EFS_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide).

Author: ```suggestion
16. To use **EFS**, you can either select **Use existing EFS file system** and specify an existing EFS instance, or select **Create new EFS file system** to create one. If you intend to use the EFS file system as your work directory, you will need to specify `<your_EFS_mount_path>/work` in the **Pipeline work directory** field(step 8 of this guide).
```

>     - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. This is the path where the EFS volume is accessible to the compute environment. For simplicity, we advise that you specify `/mnt/efs` as the EFS mount path.

Author: ```suggestion
    - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. This is the path where the EFS volume is accessible to the compute environment. For simplicity, we advise that you use `/mnt/efs` as the EFS mount path.
```

>     - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. This is the path where the EFS volume is accessible to the compute environment. If left blank, the mount path defaults to the pipeline work directory root path.

Author: ```suggestion
    - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. This is the path where the EFS volume is accessible to the compute environment. For simplicity, we advise that you specify `/mnt/efs` as the EFS mount path.
```

>     - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. If left blank, the mount path defaults to the pipeline work directory root path. This must be a valid mount path for an existing EFS file system that is accessible to your environment.

Reviewer: For me, this sentence _"This must be a valid mount path for an existing EFS file system that is accessible to your environment."_ is a bit confusing. I think that here we just need to stress that this path is where the EFS volume will be available inside the compute containers.

>     - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. If left blank, the mount path defaults to the pipeline work directory root path. This must be a valid mount path for an existing EFS file system that is accessible to your environment.

Author: ```suggestion
    - To use an existing EFS file system, enter the **EFS file system id** and **EFS mount path**. This is the path where the EFS volume is accessible to the compute environment. If left blank, the mount path defaults to the pipeline work directory root path.
```

Reviewer 2: IMO the generic text provides less useful guidance. I'm also not 100% that leaving it blank defaults to the work dir root path (see the support ticket Abhinav and I referenced - I seem to recall the default mount being something else).

>     - To create a new EFS file system, enter the **EFS mount path**. If left blank, the pipeline work directory will be used as the new EFS file system mount path. 

Author: ```suggestion
    - To create a new EFS file system, enter the **EFS mount path**. We advise that you specify `/mnt/efs` as the EFS mount path.
```

17. To use **FSx for Lustre**, you can either select **Use existing FSx file system** and specify an existing FSx instance, or select **Create new FSx file system** to create one.

> - To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. The FSx mount path is the path where the FSx volume is accessible to the compute environment. For simplicity, we advise that you specify `/mnt/fsx` as the FSx mount path.

Author: ```suggestion
- To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. The FSx mount path is the path where the FSx volume is accessible to the compute environment. For simplicity, we advise that you use `/mnt/fsx` as the FSx mount path.
```

> 17. To use **FSx for Lustre**, you can either select **Use existing FSx file system** and specify an existing FSx instance, or select **Create new FSx file system** to create one. For new and existing FSx file systems, you will need to specify `<your_FSx_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide). 

Author: ```suggestion
17. To use **FSx for Lustre**, you can either select **Use existing FSx file system** and specify an existing FSx instance, or select **Create new FSx file system** to create one. If you intend to use the FSx file system as your work directory, you will need to specify `<your_FSx_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide). 
```

> - To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. The FSx mount path is the path where the FSx volume is accessible to the compute environment. If left blank, the mount path defaults to the pipeline work directory root path. 

Author: ```suggestion
- To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. The FSx mount path is the path where the FSx volume is accessible to the compute environment. For simplicity, we advise that you specify `/mnt/fsx` as the FSx mount path.
```

> - To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. If left blank, the mount path defaults to the pipeline work directory root path. This must be a valid mount path for an existing FSx file system that is accessible to your environment.

Reviewer: Same comment here at _"This must be a valid mount path for an existing FSx file system that is accessible to your environment."_

> - To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. If left blank, the mount path defaults to the pipeline work directory root path. This must be a valid mount path for an existing FSx file system that is accessible to your environment.

Author: ```suggestion
- To use an existing FSx file system, enter the **FSx DNS name** and **FSx mount path**. The FSx mount path is the path where the FSx volume is accessible to the compute environment. If left blank, the mount path defaults to the pipeline work directory root path. 
```

> 17. To use **FSx for Lustre**, you can either select **Use existing FSx file system** and specify an existing FSx instance, or select **Create new FSx file system** to create one. If you intend to use the FSx file system as your work directory, you will need to specify `<your_FSx_mount_path>/work` as your **Pipeline work directory** (step 8 of this guide). 

Author: ```suggestion
17. To use **FSx for Lustre**, you can either select **Use existing FSx file system** and specify an existing FSx instance, or select **Create new FSx file system** to create one. If you intend to use the FSx file system as your work directory, you will need to specify `<your_FSx_mount_path>/work` in the **Pipeline work directory** field (step 8 of this guide). 
```

> - To create a new FSx file system, enter the **FSx size** (in GB) and the **FSx mount path**. If left blank, the pipeline work directory will be used as the new FSx file system mount path. 

Author: ```suggestion
- To create a new FSx file system, enter the **FSx size** (in GB) and the **FSx mount path**. We advise that you specify `/mnt/fsx` as the FSx mount path.
```
